specs/Standards/NonFungibleToken/Core.md

```diff
@@ -162,7 +160,7 @@ The following behavior is required, but contract authors may name this function
 //
 // Returns true if token was successfully transferred to `receiver_id`.
 function nft_resolve_transfer(
-  owner_id: string,
+  previous_owner_id: string,
```

Comment:
Given that this was [implemented and de-facto standardized via near-sdk-rs](https://github.com/near/near-sdk-rs/blob/a903f8c44a7be363d960838d92afdb22d1ce8b87/near-contract-standards/src/non_fungible_token/core/resolver.rs#L36), we should just merge it